draft-ietf-httpbis-priority.md

```diff
@@ -755,6 +755,23 @@ practical barriers to determining reasonable stream concurrency limits depending
 on the information that is available to them from the QUIC transport layer.
 TODO: so what can we suggest?
 
+# Retransmission Considerations in QUIC Transport
```

Comment:
I think this section would be better placed after the `Server Scheduling` section. We could then rename it to `Retransmission Scheduling`.